### PR TITLE
Remove diagnostics and lowercase buildtools package name

### PR DESCRIPTION
**Pull request by piotrpMSFT**

### CODE REVIEW

**Comment by piotrpMSFT** [comment]
@dotnet-bot test this please.

File: init-tools.sh [hunk 1]
```diff
@@ -16,13 +16,12 @@ DIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
...
 $DIR/scripts/obtain/dotnet-install.sh --channel $CHANNEL --verbose
 
-__init_tools_log=$DIR/init-tools.log
 __BUILD_TOOLS_CLI_VERSION=$(cat "$DIR/BuildToolsCliVersion.txt")
 __BUILD_TOOLS_DIR=$DIR/build_tools
 __BUILD_TOOLS_CLI_DIR=$__BUILD_TOOLS_DIR/dotnetcli/
 __BUILD_TOOLS_SOURCE=https://dotnet.myget.org/F/dotnet-buildtools/api/v3/index.json
 __BUILD_TOOLS_PACKAGE_VERSION=$(cat $DIR/BuildToolsVersion.txt)
-__BUILD_TOOLS_PATH=$NUGET_PACKAGES/Microsoft.DotNet.BuildTools/$__BUILD_TOOLS_PACKAGE_VERSION/lib
+__BUILD_TOOLS_PATH=$NUGET_PACKAGES/microsoft.dotnet.buildtools/$__BUILD_TOOLS_PACKAGE_VERSION/lib
```
Comment (eerhardt) [inline init-tools.sh]: Would it make sense to do this on Windows for consistency sake?

File: init-tools.ps1 [hunk 2]
```diff
@@ -50,10 +49,10 @@ if (!(Test-Path "$BUILD_TOOLS_PATH"))
 
 # Restore build tools
 $args="restore $PROJECT_JSON_FILE --packages $PACKAGES_DIR --source $BUILD_TOOLS_SOURCE"
-Start-Process -FilePath $DOTNET_EXE_CMD -ArgumentList $args -Wait -RedirectStandardOutput $INIT_TOOLS_LOG -NoNewWindow
+Start-Process -FilePath $DOTNET_EXE_CMD -ArgumentList $args -Wait -NoNewWindow
```
Comment (eerhardt) [inline init-tools.ps1]: /cc @sokket

**Comment by livarcocc** [comment]
:shipit:

**Comment by eerhardt** [comment]
Is this change still happening? Any update?

**Pull request closed without merging**